tooling/bundler/src/bundle/windows/nsis.rs

```diff
@@ -100,6 +100,13 @@ fn get_and_extract_nsis(nsis_toolset_path: &Path, _tauri_tools_path: &Path) -> c
   let data = download(NSIS_APPLICATIONID_URL)?;
   info!("extracting NSIS ApplicationID plugin");
   extract_zip(&data, &nsis_plugins)?;
+  info!("extracted NSIS ApplicationID plugin");
+
+  let nsis_plugins_x86 = nsis_plugins.join("x86-unicode");
+  if !nsis_plugins_x86.exists() {
+    create_dir_all(nsis_plugins_x86)?;
+  }
```

Comment:
```suggestion
  create_dir_all(nsis_plugins.join("x86-unicode"))?;
```
No need to check if it exists first imo. create_dir_all won't delete anything if a dir already exists.

Reply:
thank you, I'm not a Rust developer .... yet :)